packages/zosmf/src/ZosmfSession.ts

```diff
@@ -115,6 +115,17 @@ export class ZosmfSession {
         group: ZosmfSession.ZOSMF_CONNECTION_OPTION_GROUP
     };
 
+    /**
+     * Option used to specify HTTP or HTTPS Protocol
+     */
+    public static ZOSMF_OPTION_PROTOCOL: ICommandOptionDefinition = {
+        name: "protocol",
+        aliases: ["o"],
+        description: "The protocol used (HTTP or HTTPS)",
+        type: "string",
+        group: ZosmfSession.ZOSMF_CONNECTION_OPTION_GROUP
+    };
```

Comment:
Should we define "http" and "https" as the only allowed values for the --protocol command option?

Reply:
I don't see a problem in doing so. 😋